bin/resolve.sh

```diff
@@ -2,12 +2,21 @@
 
 # Source env settings
 . .env
+source bin/helper.sh
 
 # NeoGo binary path.
 NEOGO="${NEOGO:-docker exec -it morph_chain neo-go}"
 # NNS contract script hash
-NNS_ADDR=`curl -s --data '{ "id": 1, "jsonrpc": "2.0", "method": "getcontractstate", "params": [1] }' http://morph-chain.${LOCAL_DOMAIN}:30333/ | jq -r '.result.hash'`
+output=$(curl -s --data '{ "id": 1, "jsonrpc": "2.0", "method": "getcontractstate", "params": [1] }' \
+	"http://morph-chain.${LOCAL_DOMAIN}:30333/") \
+	|| die "Cannot fetch NNS contract state"
+ 
+NNS_ADDR=$(jq -r '.result.hash' <<< "$output") \
+	|| die "Cannot parse NNS contract hash $NNS_ADDR"
```

Comment:
is it expected to work correctly? i mean printing `NNS_ADDR` in err after we failed to find it

Reply:
/cc @fyrchik

Reply:
how about adding ": " to the output?

Reply:
fixed.